airbyte-cdk/python/airbyte_cdk/sources/declarative/create_partial.py

```diff
@@ -40,7 +38,7 @@ def newfunc(*fargs, **fkeywords):
         fully_created = _create_inner_objects(all_keywords, options)
 
         # interpolate the parameters
-        interpolated_keywords = InterpolatedMapping(fully_created, interpolation).eval(config, **{"options": options})
+        interpolated_keywords = InterpolatedMapping(fully_created).eval(config, **{"options": options})
```

Comment:
we always use a `JinjaInterpolation`. not passing it as argument makes the interface simpler